script/hassfest/coverage.py

```diff
@@ -20,12 +21,8 @@
 # They were violating when we introduced this check
 # Need to be fixed in a future PR.
 ALLOWED_IGNORE_VIOLATIONS = {
-    ("doorbird", "logbook.py"),
-    ("elkm1", "scene.py"),
-    ("fibaro", "scene.py"),
-    ("lcn", "scene.py"),
     ("lutron", "scene.py"),
-    ("tuya", "scene.py"),
+    ("stream", "diagnostics.py"),
```

Comment:
I was able to remove `doorbird`, `elkm1`, `fibaro`, `lcn` and `tuya`
But sadly I had to add `stream` in here.

Reply:
After further checking - it seems that stream is not the only component mis-behaving regarding `diagnostics.py`

Reply:
Do all of these not have diagnostics tests or do they have a surplus ignore item for the diagnostics module?

Reply:
I did not check them individually.
I thought it would be best to first prevents new ones coming in, and then see which can be removed afterwards.

Reply:
I have just checked: none of the components being added has a corresponding `test_diagnosticss.py` file.